docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1369,6 +1369,22 @@ A value of `0` means "never". The default value corresponds to 30 days.
 
 Default: `2592000` (30 days).
 
+## database_catalog_drop_error_cooldown_sec
+
+In case of a failed table drop, ClickHouse will wait for this time-out before retrying the operation.
+
+Type: [`UInt64`](../../sql-reference/data-types/int-uint.md)
+
+Default: `0`
```

Comment:
```
┌─name─────────────────────────────────────┬─value─┬─default─┬
│ database_catalog_drop_error_cooldown_sec │ 5     │ 5       │
└──────────────────────────────────────────┴───────┴─────────┴
```

Reply:
One other easy way to check for the default values for the various server parameters is by querying the `system.server_settings` table (which is what I usually do): https://pastila.nl/?03558f24/36d6cc5a5c0d926dcc5ae2950de6d96b#CCTrDfENtI2GXI16EcO/yA==